akka-http/src/main/scala/akka/http/javadsl/server/Rejections.scala

```diff
@@ -57,6 +57,16 @@ trait MissingQueryParamRejection extends Rejection {
   def parameterName: String
 }
 
+/**
+ * Rejection created by parameter filters.
+ * Signals that the request was rejected because a query parameter value was not equal to required one.
+ */
+trait InvalidRequiredValueForQueryParamRejection extends Rejection {
```

Comment:
Perhaps mark with `@DoNotInherit`?

Reply:
I can mark it as a `@DoNotInherit`, but what with the rest of a traits in Rejections file? I think we should mark all traits or none to keep a consistency.

Reply:
In my opinion, I would put `@DoNotInherit` to all of them